includes/data/connection/class-product-connection-resolver.php

```diff
@@ -21,6 +21,8 @@
 
 /**
  * Class Product_Connection_Resolver
+ *
+ * @deprecated v0.10.0
```

Comment:
@kidunot89 Perhaps we should provide some guidance as to what to use instead.

`@deprecated since 0.10.0. Use PostObjectConnectionResolver instead.`